platform/engineering/codeowners.md

```diff
@@ -1,23 +1,28 @@
 # Code owners
 
-## Overview 
+## Overview
 
-This document covers how code owners are used to enforce required reviews. The intended audience for this is leads and developers on VFS teams who want to use code owners to enforce required reviews on their applications. 
+This document covers how code owners are used to enforce required reviews. The intended audience for this is leads and developers on VFS teams who want to use code owners to enforce required reviews on their applications.
 
-## Code owners on `vets-website`
+## VSP Code Owners
 
-VSP uses GitHub's [code owners](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) feature to trigger required reviews. A code owner is a GitHub [team](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-teams) that must approve a change before it can be merged. 
+VSP uses GitHub's [code owners](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) feature to trigger required reviews. A code owner is a GitHub [team](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-teams) that must approve a change before it can be merged.
 
-By default, [`frontend-review-group`](https://github.com/orgs/department-of-veterans-affairs/teams/frontend-review-group) is the code owner of `vets-website`. VSA teams can optionally create a GitHub team under the [Department of Veteran Affairs GitHub organization](https://github.com/orgs/department-of-veterans-affairs/teams) and assign that team as a code owner of [application folders](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications). You can create a team by clicking on the **New team** button on [this page](https://github.com/orgs/department-of-veterans-affairs/teams)
+By default, [`frontend-review-group`](https://github.com/orgs/department-of-veterans-affairs/teams/frontend-review-group) is the code owner of `vets-website` and [`backend-review-group`](https://github.com/orgs/department-of-veterans-affairs/teams/backend-review-group) is the code owner of `vets-api`.
 
-**This will remove `frontend-review-group` as a required reviewer when changes are made to those folders.** 
+## VSA Code Owndes
+VSA teams can optionally create a GitHub team under the [Department of Veteran Affairs GitHub organization](https://github.com/orgs/department-of-veterans-affairs/teams) by clicking on the **New team** button. This must be done for the team to become a code owner.
```

Comment:
I think there's some naming guidelines for how to name GH teams

Reply:
Found it! Might be helpful to link the naming guidelines here https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/vsp_user_managment_process.md#creating-a-github-team

Reply:
@erluti My first instinct is to say leave this document with "optionally create a GitHub team", but link to the WIP doc for naming guidelines as @LindseySaari suggested. I'd leave the other doc WIP for now. I have not yet heard whether we've decided to make creating a GitHub team mandatory. We can follow-up with changes if needed.

Reply:
Thanks so much for the feedback!